### PR TITLE
net-mail/mpop: bump to version 1.4.16

### PR DESCRIPTION
**Pull request by kernlpanic**
Thanks-To: Herbert Wantesh <rauchwolke@gmx.net>
Closes: https://bugs.gentoo.org/733206
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Dennis Eisele <kernlpanic@dennis-eisele.de>

### CODE REVIEW

File: net-mail/mpop/mpop-1.4.16.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+EAPI=8
+
+DESCRIPTION="A small, fast, and portable POP3 client"
+HOMEPAGE="https://marlam.de/mpop/"
+SRC_URI="https://marlam.de/mpop/releases/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="gnutls idn gnome-keyring nls sasl ssl vim-syntax"
```
Comment (thesamesam) [inline net-mail/mpop/mpop-1.4.16.ebuild:13]: What about https://bugs.gentoo.org/733206#c2?

i.e. Rip out OpenSSL support maybe?

File: net-mail/mpop/mpop-1.4.16.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	nls? ( virtual/libintl )
+	sasl? ( virtual/gsasl )
+	ssl? (
+		gnutls? ( net-libs/gnutls:0=[idn?] )
+		!gnutls? (
+			dev-libs/openssl:0=
+		)
+	)"
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )
```
Comment (thesamesam) [inline net-mail/mpop/mpop-1.4.16.ebuild:27]: pkgconfig is definitely BDEPEND here and gettext probably is

File: net-mail/mpop/mpop-1.4.16.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	virtual/pkgconfig"
+
+REQUIRED_USE="gnutls? ( ssl )"
+
+DOCS="AUTHORS ChangeLog NEWS NOTES README THANKS"
```
Comment (thesamesam) [inline net-mail/mpop/mpop-1.4.16.ebuild:32]:
```suggestion
DOCS=( AUTHORS ChangeLog NEWS NOTES README THANKS )
```

**Review by thesamesam — CHANGES_REQUESTED** [review]

**Comment by kernlpanic** [comment]
Thanks for the Feedback. I hope everything is ok now. I wasn't quite sure with the dependencies but cross-building worked for me.

**Pull request closed without merging**